index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <header class="app-header">
+            <div class="logo"><p class="text_logo">My Logo</p></div>
+            <nav class="nav-bar">
+                 <div class="nav">
+                    <img class="menu" src="Union.png" alt="menu">
+                </div>  
+            </nav>
+        </header>
+        <div class="content">
+            <h2 class="salute">I'm Alexandra. Glad to see you!</h2>
+            <p class ="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <p class="lets_connect">LET’S CONNECT</p>
+            <div class="social-links">
+                <a class="social" href="#"><img src="SocialMediaIcons\Twitter_icon.png" alt="Twitter_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Linkedin_icon.png" alt="Linkedin_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\M_icon.png" alt="M"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Github_icon.png" alt="Github_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Hand_icon.png" alt="Hand_icon"></a>
+            </div>
+        </div>
+    </div>
+    <!--<img id="Rectangle7" src="GeometricFigures\Rectangle 7.png">
+    <img id="Ellipse5" src="GeometricFigures\Ellipse 5.png">
+    <img id="Rectangle5" src="GeometricFigures\Rectangle 5.png">
+    <img id="Ellipse4" src="GeometricFigures\Ellipse 4.png">
+    <img id="Ellipse13" src="GeometricFigures\Ellipse 13.png">
+    <img id="Ellipse12" src="GeometricFigures\Ellipse 12.png">
+    <img id="Rectangle6" src="GeometricFigures\Rectangle 6.png">
+    <img id="Ellipse11" src="GeometricFigures\Ellipse 11.png">
+    <img id="Ellipse8" src="GeometricFigures\Ellipse 8.png">
+    <img id="Ellipse14" src="GeometricFigures\Ellipse 14.png">
+    <img id="Ellipse15" src="GeometricFigures\Ellipse 15.png">
+    <img id="Ellipse3" src="GeometricFigures\Ellipse 3.png">
+    <img id="Rectangle4" src="GeometricFigures\Rectangle 4.png">-->

```

Comment:
- [ ] Please remove all the commented code that you are not using for the best practices of HTML & CSS, we use commented code to describe what a section of code does.